client/js/jsonedit.js

```diff
@@ -646,33 +650,13 @@ function displayComputeOps() {
 }
 
 function jeAddCSScommands() {
-  for(const css of [ 'border: 1px solid black', 'background: white', 'font-size: 30px', 'color: black' , 'border-radius: 100%' ]) {
```

Comment:
@ArnoldSmith86  is this yours? why remove custom css per type? this also seen not to be working

Reply:
Oh, I didn't notice they had different contexts. I'll fix that. I just didn't want three times the same code.